src/components/ec-currency-input/ec-currency-input.story.js

```diff
@@ -43,8 +43,8 @@ stories
       onChange: action('change'),
     },
     template: `
-      <div style="max-width: 600px; margin: 60px auto">
-        <ec-currency-input v-model="value" :currencies="currencies" v-bind="$props" @change="onChange" class="ec-mt--20 ec-mb--20" />
+      <div class="tw-my-64 tw-mx-auto" style="max-width: 600px;">
```

Comment:
You can also replace the max-width: 600px with: `tw-max-w-screen-sm`. It's not exactly the same but we can get rid of the style. 

But I wonder that visual regression tests didn't fail after this change.

Reply:
will change it and update regression tests